### PR TITLE
Add mobile support for table views of the Subscription and Install plans

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1613669

### CODE REVIEW

File: frontend/public/components/cloud-services/subscription.tsx [hunk 1]
```diff
@@ -15,11 +15,11 @@ import { createSubscriptionChannelModal } from '../modals/subscription-channel-m
...
 export const SubscriptionHeader: React.SFC<SubscriptionHeaderProps> = (props) => <ListHeader>
-  <ColHead {...props} className="col-md-3" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-md-2" sortField="metadata.namespace">Namespace</ColHead>
-  <ColHead {...props} className="col-md-2">Status</ColHead>
-  <ColHead {...props} className="col-md-2">Channel</ColHead>
-  <ColHead {...props} className="col-md-2">Approval Strategy</ColHead>
+  <ColHead {...props} className="col-xs-6 col-sm-4 col-md-3" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-xs-6 col-sm-4 col-md-2" sortField="metadata.namespace">Namespace</ColHead>
+  <ColHead {...props} className="hidden-xs col-sm-3 col-md-3">Status</ColHead>
+  <ColHead {...props} className="hidden-xs hidden-sm col-sm-3 col-md-2">Channel</ColHead>
```
Comment (spadgett) [inline frontend/public/components/cloud-services/subscription.tsx]: Doesn't col-sm-* add up to more than 12?

**Comment by sg00dwin** [comment]
updated

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Pull request merged**